src/deadify.js

```diff
@@ -1,7 +1,7 @@
 (() => {
 	function getWeek() {
 		// The uni week without factoring out holidays/breaks
-		let weekNum = getWeekOfYear(new Date()) - 7;
+		let weekNum = getWeekOfYear(new Date()) - 26;
 
 		// Mid-term break
 		if (weekNum > 6 && weekNum < 9) {
```

Comment:
You also need to adjust this for the mid tri break.

Reply:
The T2 mid-term break is two weeks long when weekNum is 7 and 8, same as T1 mid-term break, so I'm not sure it needs adjusting this time?

Reply:
Sweet